modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -71,8 +71,14 @@ Defaults to 1 second.
 == Disabling the Salt Mine
 
 In older versions, {productname} used a tool called Salt mine to check minion availability.
-With the introduction of more efficient tools in {productname}{nbsp}3.2, it is no longer required.
+Salt mine would ping the server every hour, which created significant load.
+With the introduction of more efficient tools in {productname}{nbsp}3.2, Salt mine is no longer required.
```

Comment:
maybe we could use "With the introduction of a more efficient mechanism" instead of "more efficient tools"